### PR TITLE
vli-usbhub: Add support for updating V2 devices

### PR DESCRIPTION
**Pull request by hughsie**

### CODE REVIEW

**Comment by hughsie** [comment]
I got hardware for this a few days ago. I fixed up the various bugs that were stopping the flash working and it now updates V2 hardware successfully. Slightly-out-of-date video here: https://people.freedesktop.org/~hughsient/temp/VL817.gif

It still looks good to you?

File: plugins/vli-usbhub/fu-vli-usbhub-device.c [hunk 1]
```diff
@@ -909,12 +1308,12 @@ fu_vli_usbhub_device_init (FuVliUsbhubDevice *self)
...
 	self->spi_cmd_read_status	= 0x05;
 	self->spi_cmd_write_en		= 0x06;
+	self->spi_cmd_sector_erase	= 0x20;
 	self->spi_cmd_chip_erase	= 0x60;
 	self->spi_cmd_read_id		= 0x9f;
 	self->spi_cmd_read_id_sz = 2;
 	fu_device_add_icon (FU_DEVICE (self), "audio-card");
-	fu_device_set_summary (FU_DEVICE (self), "Expand a single USB port into several");
-	fu_device_set_install_duration (FU_DEVICE (self), 3); /* seconds */
+	fu_device_set_install_duration (FU_DEVICE (self), 15); /* seconds */
```
Comment (superm1) [inline plugins/vli-usbhub/fu-vli-usbhub-device.c]: Was v1 wrong?

Comment (hughsie) [reply plugins/vli-usbhub/fu-vli-usbhub-device.c]: Hmm, good question. I'll test.

Comment (hughsie) [reply plugins/vli-usbhub/fu-vli-usbhub-device.c]: 10s for V1, 15s for V2. Patch updated.

**Pull request merged**